ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -101,12 +101,6 @@ export const toolTipText = {
     publicly available <i>All of Us</i> website (https://www.researchallofus.org/) to inform our
     participants and other stakeholders about what kind of research their data is being used
     for.</div>,
-  reviewRequest: <div>If you are concerned that your research may be stigmatizing to a particular
```

Comment:
Moved this text to be always displayed, as per mock: https://projects.invisionapp.com/d/#/console/12820961/367391162/preview